scripts/changelog.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
I wrote one out for `release.sh` at shx to begin with. We could also do one for the changelog. (let me know if you would prefer to not have `release.sh` transpiled to JS).

I can work on translating `changelong.sh` as well or instead.